### PR TITLE
Bugfix/typeform - Fixes for bugs reported via Typeform

### PR DESCRIPTION
**Pull request by jeesunikim**
Fixes for bugs reported via Typeform

* [Subscribe Button Overlaps](https://app.asana.com/0/1119309091112078/1173687524197297)
* [Triple Scrollbar on Mac and Linux](https://app.asana.com/0/1119309091112078/1173720962232793)

### CODE REVIEW

File: src/basics/GlobalStyles.js [hunk 1]
```diff
@@ -37,6 +37,11 @@ export const GlobalStyles = createGlobalStyle`
   twitter-widget {
     margin: auto;
   }
+  ::-webkit-scrollbar {
```
Comment (vcarl) [inline src/basics/GlobalStyles.js:40]: Verrrry nervous about globally hiding scrollbars. I notice that the entire page scrolls a little bit, I think there's some extra height coming in from the sidebar

Comment (vcarl) [reply src/basics/GlobalStyles.js:40]: I looked at this earlier but hadn't committed a fix, I think `SideNavContainer` has the wrong height. It's doing `calc(100vh - 3.75rem)` but the logo section is 72px = 4.5rem.

Comment (jeesunikim) [reply src/basics/GlobalStyles.js:40]: How about adding this to the JSON responses component and I will find the extra height?

Comment (vcarl) [reply src/basics/GlobalStyles.js:40]: This could def make sense on those docs elements

Comment (jeesunikim) [reply src/basics/GlobalStyles.js]: Updated!

**Pull request merged**